### PR TITLE
Added news article "Projects Announced for GSoC 2022"

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: content/news/XXXX-XX-XX-projects-announced-for-gsoc-2022.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+title: Projects Announced for GSoC 2022
+authors: Daniel Schürmann
+tags: gsoc, gsoc-2022
+status: draft
+comments: no
+
+Google has accepted two [Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/)new contributor's projects for Mixxx. GSoC is a stipend program that allows new contributors to work Mixxx on their own ~175 hours or ~350 hours project.
+
+This year Fatih Emre YILDIZ and David Chocholatý have been successful with their applications. Congratulations!
```
Comment (Swiftb0y) [inline content/news/XXXX-XX-XX-projects-announced-for-gsoc-2022.md]: id maybe link their github profiles here, but other than that, lgtm

Comment (Swiftb0y) [reply content/news/XXXX-XX-XX-projects-announced-for-gsoc-2022.md]:
```suggestion
This year [Fatih Emre YILDIZ](https://github.com/fatihemreyildiz) and [David Chocholatý](https://github.com/davidchocholaty) have been successful with their applications. Congratulations!
```

**Review by Swiftb0y — APPROVED** [review]
Pre-commit is still complaining, but I'll merge now so it gets published ASAP. I'd appreciate if you could fix the remaining issues once you find the time.

**Comment by daschuer** [comment]
the pending pre-commit issue is fixed here: https://github.com/mixxxdj/website/pull/269

**Pull request merged**